app/views/compute_resources_vms/form/azurerm/_network.html.erb

```diff
@@ -1,5 +1,7 @@
 <%= selectable_f f, :network, [],
-                 { :include_blank => _('Please first select an image and an Azure region')},
+                 { :include_blank => _('Please first select an image and an Azure region'),
+                   :selected => nil
```

Comment:
I didn't get the purpose of `selected => nil`. Any specific reason behind this?

Reply:
This `nil` was explicitly set here since `selectable_f` expected some value for selected attribute here. But, since we want to set the value selected by user at the time of host creation or profile creation, hence it should be set to that value and not nil. Handling this in #43 (WIP). So, closing this PR in favor of #43